app/join/components/JoinContext.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+import { FormProvider, useForm, useFormContext } from 'react-hook-form';
+
+import type { SignUpState } from '../type';
+import type { StrictPropsWithChildren } from '@/types';
+
+const defaultValues: SignUpState = {
```

Comment:
defaultValues가 이미 타입을 알고 있어서 타입 선언하면 undefined같은거 못하게 됩니다! 여기서는 타입 선언하지 않도록 해주세요
```suggestion
const defaultValues = {
```

Reply:
d144be6